package.json

```diff
@@ -43,11 +43,11 @@
     "esvalid": "1.1.0",
     "fs-readdir-recursive": "0.1.0",
     "jshint": "2.5.11",
-    "lodash": "2.4.1",
+    "lodash": "3.0.0",
```

Comment:
No way, didn't meant to imply it was, never had any issues with lodash! It was facebook/regenerator#164